startlxqt.in

```diff
@@ -26,9 +26,11 @@ export XDG_DATA_DIRS
 if [ -z "$XDG_CONFIG_DIRS" ]; then
     export XDG_CONFIG_DIRS="@PREDEF_XDG_CONFIG_DIRS@"
 else
-    if ! contains "$XDG_CONFIG_DIRS" '@LXQT_ETC_XDG_DIR@'; then
-        XDG_CONFIG_DIRS="$XDG_CONFIG_DIRS:@LXQT_ETC_XDG_DIR@"
-    fi
+    for directory in "/etc" "@LXQT_ETC_XDG_DIR@" "/usr/share"; do
```

Comment:
So in this way can drop the logic around `PREDEF_XDG_CONFIG_DIRS` and just leave this (list) check/add?